File: src/main/java/cube8540/oauth/authentication/credentials/oauth/scope/domain/OAuth2ScopeRepository.java
package cube8540.oauth.authentication.credentials.oauth.scope.domain;

import cube8540.oauth.authentication.credentials.AuthorityCode;
import org.springframework.data.jpa.repository.JpaRepository;

public interface OAuth2ScopeRepository extends JpaRepository<OAuth2Scope, AuthorityCode> {

    Long countByCode(AuthorityCode scopeId);

}

File: src/main/kotlin/cube8540/oauth/authentication/credentials/oauth/scope/domain/OAuth2ScopeRepository.kt
package cube8540.oauth.authentication.credentials.oauth.scope.domain

import cube8540.oauth.authentication.credentials.AuthorityCode
import org.springframework.data.jpa.repository.JpaRepository

interface OAuth2ScopeRepository: JpaRepository<OAuth2Scope, AuthorityCode> {

    fun countByCode(scopeId: AuthorityCode): Long
}